### PR TITLE
Add SDV IVT to suite

### PR DESCRIPTION
**Pull request by Mark-J-Lawrence**
Adding the SDV Manager IVT to the IVT test suite.

Provides the required CPS properties for the test to run.

### CODE REVIEW

File: galasa-inttests-parent/dev.galasa.inttests/src/main/java/dev/galasa/inttests/sdv/AbstractSDVLocal.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    getEcosystem().setCpsProperty("cicsts.dse.tag.A.applid", cics.getApplid());
+    getEcosystem().setCpsProperty("cicsts.dse.tag.A.version", cics.getVersion().toString());
+    getEcosystem().setCpsProperty("cicsts.default.logon.initial.text", "HIT ENTER FOR LATEST STATUS");
+    getEcosystem().setCpsProperty("cicsts.default.logon.gm.text", "******\\(R)");
+
+    // SDV CPS props
+    getEcosystem().setCpsProperty("sdv.roleTag.R1.role", "TELLER");
+    getEcosystem().setCpsProperty("sdv.zosImage." + cics.getZosImage().getImageID() + ".role.TELLER.credTags", "???");
+    getEcosystem().setCpsProperty("sdv.cicsTag.A.port", "???");
+    getEcosystem().setCpsProperty("sdv.cicsTag.A.hlq", "???");
```
Comment (Mark-J-Lawrence) [inline galasa-inttests-parent/dev.galasa.inttests/src/main/java/dev/galasa/inttests/sdv/AbstractSDVLocal.java:38]: Hi @jadecarino, these are the CPS properties I've trouble filling in for the SDV IVT. 

I need a credentials tag, probably the one for the user running the test would do...
A port number for the SDC service to run on
and a HLQ for the CICS install.....not sure if `ANTZ.CICS.TS.DEV.INTEGRAT` would be safe to put here?

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by Mark-J-Lawrence** [comment]
These code changes fail when building locally currently with 
```
[ERROR] Failed to execute goal on project dev.galasa.inttests.obr: Could not resolve dependencies for project dev.galasa:dev.galasa.inttests.obr:galasa-obr:0.35.0: Failed to collect dependencies at dev.galasa:dev.galasa.inttests:jar:0.35.0: Failed to read artifact descriptor for dev.galasa:dev.galasa.inttests:jar:0.35.0: The following artifacts could not be resolved: dev.galasa:galasa-bom:pom:0.35.0 (absent): dev.galasa:galasa-bom:pom:0.35.0 was not found in https://repo.maven.apache.org/maven2/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of maven.central has elapsed or updates are forced -> [Help 1]
```

File: galasa-inttests-parent/dev.galasa.inttests/src/main/java/dev/galasa/inttests/sdv/AbstractSDVLocal.java (hunk 1 above)
Comment (Mark-J-Lawrence) [reply galasa-inttests-parent/dev.galasa.inttests/src/main/java/dev/galasa/inttests/sdv/AbstractSDVLocal.java:38]: another thing to consider is....do we want to make these values public, in this file?

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

File: galasa-inttests-parent/dev.galasa.inttests/src/main/java/dev/galasa/inttests/sdv/AbstractSDVLocal.java (hunk 1 above)
Comment (jadecarino) [reply galasa-inttests-parent/dev.galasa.inttests/src/main/java/dev/galasa/inttests/sdv/AbstractSDVLocal.java:39]: @Mark-J-Lawrence Hi Mark. So just to clarify, do these CPS properties for your manager definitely have to be set in the child ecosystem that's spawned to run the IVT, or could they go just in our prod Galasa ecosystem's CPS properties?

Comment (Mark-J-Lawrence) [reply galasa-inttests-parent/dev.galasa.inttests/src/main/java/dev/galasa/inttests/sdv/AbstractSDVLocal.java:39]: @jadecarino No they definitely don't, if they already exist as part of the ecosystems CPS properties, I can remove them from here

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Review by jadecarino — APPROVED** [review]

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**